### ISSUE TITLE
How do I use pretrained model?

### ISSUE THREAD

**Issue by DepressiveDude** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#363]
Sorry if it is a dumb question, but I can't seem to figure out which models are which. I followed the instructions for installing, which includes downloading from huggingface. However, the inference voice list is empty, even if I restarted the program. I am new to this so I might be missing something, any help is appreciated.

**Comment by johndpope** [comment]
this is too hard - 

Suggest updating ui to say 

Inferencing voice:
(load models to ./pretrained/1111.pth) 


but doesn't work 

am using a extracted zip file from here
 https://huggingface.co/QuickWick/Music-AI-Voices/tree/main

**Comment by Ma5onic** [comment]
https://youtu.be/-JcvdDErkAU

**Comment by kronkinatorix** [comment]
Of course, why write anything down or have meaningful documentation I can search when I can waste even more time sitting through a fifteen minute youtube video./

Youtube is a grotesque metastasized adenoid greedily eating the internet's utility.
